src/yuzu/configuration/configure_input_player_widget.cpp

```diff
@@ -0,0 +1,1769 @@
+        DrawRoundButton(p, center + QPoint(180, -65), button_values[Y], 5, 10, Direction::Left);
+
+        // Sideview D-pad buttons
+        DrawRoundButton(p, center + QPoint(-180, 12), button_values[DLeft], 5, 10,
+                        Direction::Right);
+        DrawRoundButton(p, center + QPoint(-180, 33), button_values[DDown], 5, 10,
+                        Direction::Right);
+        DrawRoundButton(p, center + QPoint(-180, -8), button_values[DUp], 5, 10, Direction::Right);
+        DrawRoundButton(p, center + QPoint(-180, 12), button_values[DRight], 5, 10,
+                        Direction::Right);
+
+        // Sideview home and plus button
+        DrawRoundButton(p, center + QPoint(180, 60), button_values[Home], 3, 11, Direction::Left);
+        DrawRoundButton(p, center + QPoint(180, -106), button_values[Plus], 4, 7, Direction::Left,
+                        1);
+        DrawRoundButton(p, center + QPoint(180, -106), button_values[Plus], 4, 2.33f,
+                        Direction::Left, 1);
+
+        // Sideview screenshot and minus button
+        DrawRoundButton(p, center + QPoint(-180, 63), button_values[Screenshot], 3, 8,
+                        Direction::Right, 1);
+        DrawRoundButton(p, center + QPoint(-180, -106), button_values[Minus], 4, 2.66f,
+                        Direction::Right, 1);
+    }
+
+    DrawDualBody(p, center);
+
+    { // Draw joysticks
+        using namespace Settings::NativeAnalog;
+        DrawJoystick(p, center + QPointF(-65, -65) + (axis_values[LStick].value * 7), 1.62f,
+                     button_values[Settings::NativeButton::LStick]);
+        DrawJoystick(p, center + QPointF(65, 12) + (axis_values[RStick].value * 7), 1.62f,
+                     button_values[Settings::NativeButton::RStick]);
+        DrawRawJoystick(p, rect().bottomLeft() + QPointF(45, -45), axis_values[LStick].raw_value,
+                        axis_values[LStick].properties);
+        DrawRawJoystick(p, rect().bottomRight() + QPointF(-45, -45), axis_values[RStick].raw_value,
+                        axis_values[RStick].properties);
+    }
+
+    using namespace Settings::NativeButton;
+
+    // Face buttons constants
+    const QPointF face_center = center + QPoint(65, -65);
+    const int face_distance = 20;
+    const int face_radius = 10;
+    const int text_size = 10;
+
+    // Face buttons
+    p.setPen(colors.outline);
+    button_color = colors.button;
+    DrawCircleButton(p, face_center + QPoint(face_distance, 0), button_values[A], face_radius);
+    DrawCircleButton(p, face_center + QPoint(0, face_distance), button_values[B], face_radius);
+    DrawCircleButton(p, face_center + QPoint(0, -face_distance), button_values[X], face_radius);
+    DrawCircleButton(p, face_center + QPoint(-face_distance, 0), button_values[Y], face_radius);
+
+    // Face buttons text
+    p.setPen(colors.font);
+    DrawText(p, face_center + QPoint(face_distance, 0), text_size, QString::fromStdString("A"));
```

Comment:
```suggestion
    DrawText(p, face_center + QPoint(face_distance, 0), text_size, QStringLiteral("A"));
```

Ditto for the other calls